src/main/scala/dpla/ingestion3/mappers/providers/InMapping.scala

```diff
@@ -0,0 +1,140 @@
+class InMapping extends XmlMapping with XmlExtractor
+  with IngestMessageTemplates {
+
+  override def useProviderName: Boolean = true
+
+  override def getProviderName: String = "il"
+
+  // TODO: Confirm with hub that this is the correct mapping
+  override def originalId(implicit data: Document[NodeSeq]): ZeroToOne[String] =
+    extractString(data \ "header" \ "identifier")
+
+  override def dplaUri(data: Document[NodeSeq]): ZeroToOne[URI] = mintDplaItemUri(data)
+
+  override def dataProvider(data: Document[NodeSeq]): ZeroToMany[EdmAgent] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "provenance")
+      .map(nameOnlyAgent)
+
+  // TODO: In ingest1, the following are also mapped to edmRights:
+  //   "accessRights" where the value starts with "http"
+  //   Should we replicate this logic here?
+  override def edmRights(data: Document[NodeSeq]): ZeroToMany[URI] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "rights")
+      .map(URI)
+
+  override def intermediateProvider(data: Document[NodeSeq]): ZeroToOne[EdmAgent] =
+    extractString(data \ "metadata" \ "qualifieddc" \ "mediator")
+      .map(nameOnlyAgent)
+
+  override def isShownAt(data: Document[NodeSeq]): ZeroToMany[EdmWebResource] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "identifier").map(stringOnlyWebResource)
+
+  override def originalRecord(data: Document[NodeSeq]): ExactlyOne[String] = Utils.formatXml(data)
+
+  // SourceResource
+
+  override def alternateTitle(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "alternative")
+
+  // TODO: Check with hub that this mapping is still valid.
+  //  This is in ingest1 mapping, but no harvested records have contributor.
+  override def contributor(data: Document[NodeSeq]): ZeroToMany[EdmAgent] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "contributor")
+      .map(nameOnlyAgent)
+
+  override def creator(data: Document[NodeSeq]): ZeroToMany[EdmAgent] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "creator")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(nameOnlyAgent)
+
+  override def date(data: Document[NodeSeq]): ZeroToMany[EdmTimeSpan] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "created")
+      .flatMap(_.splitAtDelimiter(";"))
+      .map(stringOnlyTimeSpan)
+
+  override def description(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "description")
+
+  override def extent(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "extent")
+
+  override def format(data: Document[NodeSeq]): ZeroToMany[String] =
+    extractStrings(data \ "metadata" \ "qualifieddc" \ "medium")
+      .flatMap(_.splitAtDelimiter(";"))
+
+  // TODO: This is in the crosswalk but not in ingest1. It is also identical to format mapping.  Keep?
+  override def genre(data: Document[NodeSeq]): ZeroToMany[SkosConcept] =
```

Comment:
genre is a deprecated field in MAPv5 so this is going to be dropped in the near term anyway. I'm 50/50 on keeping it [tilt towards removing] as I do not believe `genre` is made available in the API.